homeassistant/components/zwave_js/light.py

```diff
@@ -410,18 +410,21 @@ def _get_color_values(self) -> tuple[Value | None, ...]:
     @callback
     def _calculate_color_support(self) -> None:
         """Calculate light colors."""
-        (red_val, green_val, blue_val, ww_val, cw_val) = self._get_color_values()
+        (
+            red,
+            green,
+            blue,
+            warm_white,
+            cool_white,
+        ) = self._get_color_values()
```

Comment:
```suggestion
        (red, green, blue, warm_white, cool_white) = self._get_color_values()
```

Reply:
I think I tried this but ruff formatted it back, will try again later

Reply:
Think it should work. The total line length would even be smaller than the original code. Could have been a trailing comma issue, maybe.

_If it doesn't work though, feel free to merge it as is._